dotCMS/src/main/java/com/dotcms/variant/model/AbstractVariant.java

```diff
@@ -19,10 +20,16 @@ public interface AbstractVariant extends Serializable {
     @JsonProperty("name")
     String name();
 
+    @Value.Default
     @JsonProperty("description")
-    String description();
+    default Optional<String> description() {
```

Comment:
Does this mean that the immutable will have a `Optional` attribute?

Reply:
That's correct!